eslint.config.mjs

```diff
@@ -73,7 +73,21 @@ export default [
     },
     rules: {
       'local/no-duplicate-i18n-values': 'error',
+      'unicorn/consistent-empty-array-spread': 'error',
+      'unicorn/consistent-function-scoping': 'error',
+      'unicorn/explicit-length-check': 'error',
+      'unicorn/no-instanceof-builtins': 'error',
```

Comment:
Why do we not want instanceof?

Reply:
[docs](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-instanceof-builtins.md) link to the [logic](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#instanceof_and_multiple_realms) 

for us, that's going to possibly be 
1. web stuff
2. workers vs. main thread

maybe it's overly cautious, but if the linter can change to to `typeof` etc that seems good

Reply:
> I can barely why anyone would use it

because JS isn't their primary language (think Core Java dev, OSS contributor who looks at Apex or some other language all day, etc) and they expect JS to be reasonable but it's only "mostly reasonable"